experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/LivePersonAgentBot.sln

```diff
@@ -0,0 +1,24 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We don't typically check solution files in, as all that's needed is the .csproj file. Consider removing this file.